.codeclimate.yml

```diff
@@ -0,0 +1,34 @@
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - javascript
+  eslint:
+    enabled: true
+    checks:
+      no-unused-expressions:
+        enabled: false
+  fixme:
+    enabled: true
+  nodesecurity:
+    enabled: true
+  markdownlint:
```

Comment:
Will need to update to provide better linting on markdown.  Showing line too long issues that are not valid.